manual/cli.rst

```diff
@@ -4005,6 +4021,11 @@ generate:
   lossy, so images may have artifacts. These are not usually
   noticeable to the casual observer.
 
+- ``--jpeg-quality=...``: use a lower value for JPEG quality level.
+  The default of most JPEG libraries is 75. Smaller numbers result in
```

Comment:
Should we mention that this requires --optimize-images. As it stands, on a quick read it would be reasonable to assume that one could compress jpeg images without replacing non-jpeg images.

Reply:
Actually, making it recompress existing jpeg images is a very small fix, and it seems useful since it will still only keep the new result if it shrinks the file size.

Reply:
I looked at some of the history in the meantime. I think this would make a few users happy.